project.md

```diff
@@ -58,11 +76,19 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+Though I am seeking to maintain syntax from Google and Excel, I would like to test out new design implementations which can be challenging so I think this project will require about equal parts time complexity on the language designs I am hoping to test (50%) as on the "systems" aspect and implementation (50%). If I do find that trying to create new designs demands too much implementation complexity, I will reduce the design structures I am planning to work on and I will explain the technical difficulties which prevented me from being able to design the DSL in the fashion I desired so that I do not spend a disproportionate amount of time implementing my designs but I cna still convey my concepts.
+
 ### Scope
 
 _How big an idea is this? How ambitious is this project?_
 
+I think this project is not too ambitious for the remainder of the semester that we have to work on it. However, it is a very big idea and I think that based on the experience with the PicoBot DSL experiment, I may encounter roadblocks which I am accounting for because while I would like to test out new designs, my primary goal would be to get the functionality of Excel and Google sheets so that I can address the needs I outlined of users.
+
+More clearly, I would like to: Meet the needs of users and allows for performing calculations in a manner which does not require a much different background than being able to use Excel/Google Sheets. Then getting into the challenges I have set for myself, I would like to build upon this implementation and try out new design decisions, such as allowing for argument inputting to differ from standard practices in Excel/Google Sheets, and handle errors in a more explicit manner than the current ambiguous errors users see.
```

Comment:
These are great goals. I especially see the need for great error handling that is understandable by the user.